service/src/main/java/bio/terra/landingzone/stairway/flight/create/resource/step/CreatePostgresLogSettingsStep.java

```diff
@@ -0,0 +1,60 @@
+package bio.terra.landingzone.stairway.flight.create.resource.step;
+
+import bio.terra.landingzone.library.landingzones.definition.ArmManagers;
+import bio.terra.landingzone.library.landingzones.definition.ResourceNameGenerator;
+import bio.terra.landingzone.library.landingzones.definition.factories.ParametersResolver;
+import bio.terra.stairway.FlightContext;
+import bio.terra.stairway.StepResult;
+import java.time.Duration;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class CreatePostgresLogSettingsStep extends BaseResourceCreateStep {
+  private static final Logger logger = LoggerFactory.getLogger(CreatePostgresLogSettingsStep.class);
+
+  public CreatePostgresLogSettingsStep(
+      ArmManagers armManagers,
+      ParametersResolver parametersResolver,
+      ResourceNameGenerator resourceNameGenerator) {
+    super(armManagers, parametersResolver, resourceNameGenerator);
+  }
+
+  @Override
+  public StepResult undoStep(FlightContext context) throws InterruptedException {
+    return StepResult.getStepResultSuccess();
```

Comment:
Why nothing to undo here?

Reply:
These settings will be removed together with postgres.